src/desktop/apps/auction_reaction/routes.tsx

```diff
@@ -4,16 +4,7 @@ import { routes } from "reaction/Apps/Auction/routes"
 import { stitch } from "@artsy/stitch"
 
 export const bidderRegistration = async (req, res, next) => {
```

Comment:
This function name will want to change -- something like `export const routes` (or something)